packages/manager/apps/dedicated/client/app/account/user/users/sso/add/user-users-sso-add.controller.js

```diff
@@ -0,0 +1,59 @@
+import get from 'lodash/get';
+
+export default class UserAccountUsersSSOAddCtrl {
+  /* @ngInject */
+  constructor(
+    $scope,
+    coreConfig,
+    UseraccountUsersService,
+    Alerter,
+    $translate,
+  ) {
+    this.$scope = $scope;
+    this.usersService = UseraccountUsersService;
+    this.alerter = Alerter;
+    this.$translate = $translate;
+    this.me = coreConfig.getUser();
```

Comment:
**question:** Still needed? (can't find where it's used)

Reply:
this.me removed (alerter and translate used below)